docs/en/enterprise-edition/content-collections/administration/configure-iam-security/azure-cloud-identity-inventory.adoc

```diff
@@ -0,0 +1,72 @@
+
+=== IAM Details View
+
+On the Prisma Cloud console, you can access the Identity menu from the *Assets* side panel. Start, by selecting the Assets side panel from the Alerts, Inventory or Investigate page. For example, you can select *Investigate* and run the following query to retrieve Azure cloud resources:
+
+----
+config from iam where source.cloud.type = 'AZURE'
+----
+
+Click on any asset to view more detailed information. Choose *Identity* from the side panel navigation bar to view the following IAM information:
+
+[cols="30%a,70%a"]
+|===
+
+|Group
+
+|Group membership -  Lists users assigned to the group and last access information for users 
+
+Policies - Lists policies associated with the group and last access information
+
+Granted Policies - Managed and Inline
+
+|Service Principal
+
+|Trust relationships -  Lists resources or services assigned to a role, includes last access permission data
+
+Policies - Lists policies attached to the role and last access information
+
+Trusted Entities - User, Role, SAML Provider, OIDC Provider, Lambda function, ECS, IDP   
+
+Granted Policies - Managed and Inline
+
+|Managed Identities
+
+|Role specific - Lists roles associated with a policy and last access data 
+
+Group specific - Lists groups associated with a policy and last access information 
+
+Resource specific - Lists resources (users) directly attached to the policy and last access data
+
+|===
+
+[.task]
+=== Remediate Over Privileged Access
+
+Limit over-privileged access by right-sizing permissions for Azure Groups, Service Principals and Managed Identities. The *Suggest Least Privilege Access* wizard helps you remediate overly permissive access by helping you:
+
+* Create a new policy for a Group, Service Principals or Managed Identity that includes all the permissions required by its members.
+* Repurpose existing policies that already contain the minimum required permissions for any given Group or Role. 
+
+Follow the steps below to use the *Suggest Least Privilege Access* wizard:
+
+[.procedure]
+. From the *IAM Details View* select *Suggest Least Privilege Access* from the top navigation bar. 
+. The *Suggest Least Privilege Access* wizard allows you to set the period of time after which a permission will be considered be “unused”, for a particular asset. Move the slider to any defined time limit of your choice. By default, the slider is set at 90 days and options ranging 1 day to 2 years are available. Last access days are calculated from the day IAM is enabled in your environment.
+. Next, create a customized IAM policy in your preferred output format:
+.. Select *Create New Azure Policy* to generate a file with code to create a new Custom policy, including all used permissions. Supported policy types include Managed, Custom, and Inline. The following output formats are avaiable: 
+*JSON
```

Comment:
Provide a space as after "*" , so that the bullet points are rendered correctly.

* JSON
* Terraform
* Cloud Formation